.config/CredScanSuppressions.json

```diff
@@ -3,22 +3,12 @@
  "suppressions": [
   {
     "file": [
-      "/src/Common/src/Interop/Windows/winhttp/Interop.winhttp_types.cs",
       "/src/Common/tests/System/Net/Configuration.Certificates.cs",
-      "/src/System.Data.SqlClient/src/System/Data/SqlClient/TdsEnums.cs",
-      "/src/System.Data.SqlClient/tests/FunctionalTests/DiagnosticTest.cs",
-      "/src/System.Data.SqlClient/tests/FunctionalTests/ExceptionTest.cs",
       "/src/System.Data.SqlClient/tests/FunctionalTests/SqlConnectionBasicTests.cs",
```

Comment:
we should get rid of all .cs files in this list at some point.

Reply:
That's the plan. I'm just doing it as I address issues so I know what's left to deal with.